### ISSUE TITLE
Минимизация зависимости от oscript-config

### ISSUE THREAD

**Issue by EvilBeaver** [1c-syntax/1c-syntax#306]
Предложение устанавливать oscript-config средствами самого плагина и не заставлять делать это пользователей. А можно и вовсе выяснять все что нужно средствами TypeScript.

В первом приближении: при отсутствии приложения oscript-config выполнять сначала ``opm install oscript-config`` средствами плагина. Тогда будет более клиентоориентированно.

В идеале - совсем отказаться от этого приложения, как от избыточного звена

**Comment by nixel2007** [comment]
да, есть такая фигня. я собираюсь решить эту проблему в рамках bsl language server и переключить систему автокомплита на него. первый релиз с форматированием, переведенным на рельсы LSP уже в планах. Само подключение LSP можно посмотреть в отдельном бранче в репе vsc.

**Comment by EvilBeaver** [comment]
> остальное - дело техники

Я как раз про то, чтобы "техники" было минимум. Чтобы юный падаван поставил и получил все работающее из коробки

**Comment by nixel2007** [comment]
Я тут скорее про технику тех, кто будет пилить это внутри лсп. Для падавана это все будет из коробки :)
